pkg/prowgen/prowgen_konflux.go

```diff
@@ -391,6 +391,9 @@ func generateFBCApplications(soMetadata *project.Metadata, openshiftRelease Repo
 			ComponentNameFunc: func(cfg cioperatorapi.ReleaseBuildConfiguration, ib cioperatorapi.ProjectDirectoryImageBuildStepConfiguration) string {
 				return string(ib.To)
 			},
+			FBCImages: []string{
+				fmt.Sprintf("serverless-index-%s-fbc-%s", release, v),
+			},
```

Comment:
why not using the config file ?

Reply:
The name for these indexes come only from us (not like for example from the CI config)